src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Main {
+    public static final ColorSupplier colorSupplier = new ColorSupplier();
+    public static final FigureSupplier figureSupplier = new FigureSupplier(colorSupplier);
+
+    public static void main(String[] args) {
+        InfoPrint[] figures = new InfoPrint[6];
```

Comment:
use constants instead of numbers
You need to use the `Figure`
```suggestion
        Figure[] figures = new Figure[FIGURES_NUMBER];
```